docs/release-notes/releaseposts/2024051401.md

```diff
@@ -0,0 +1,18 @@
+---
+draft: false
+date: 2024-05-14
+---
+
+# GraFx Publisher Updates
+
+![rn_icon](/assets/icon-GraFx-Publisher.svg)
+
+<!-- more -->
+
+## Improvements
+
+- Preview generation changes for GraFx Studio requirements
```

Comment:
Isn't this a bit too vague? If we add it to the release notes, I would be more specific. Maybe this:
Added "full" and "4k" preview types for GraFx Studio requirements

Reply:
On purpose. I wanted to remove it completely, but I just added in case clients have issues we or them can link it back.

I could see it going both ways to be honest. I will just push as is, but that is not because you are wrong.